lib/ansible/module_utils/powershell/Ansible.ModuleUtils.LinkUtil.psm1

```
@@ -478,12 +405,15 @@ namespace Ansible
     }
 
     $env:TMP = $remote_tmp
-    $env:TEMP = $remote_tmp
     Add-Type -TypeDefinition $link_util
     $env:TMP = $original_tmp
-    $env:TEMP = $original_temp
 
-    [Ansible.LinkUtil]::EnablePrivilege("SeBackupPrivilege")
+    Import-PrivilegeUtil
+    # enable the SeBackupPrivilege if it is disabled
+    $state = Get-AnsiblePrivilege -Name SeBackupPrivilege
+    if ($state -eq $false) {
```

silent fail if privilege not held?

---

Yep, `Get-AnsiblePrivilege` can return 3 values

* `$null`: the token doesn't have the privilege
* `$false`: the privilege is disabled
* `$true`: the privilege is enabled

It seems like the `SeBackupPrivilege` is only needed for creating symlinks while someone may use this to get info on a link from win_stat. I'm going to investigate further where exactly this privilege is needed but for now this is just a try and enable if possible.